arkouda/segarray.py

```diff
@@ -1034,6 +1035,74 @@ def to_hdf(
             ),
         )
 
+    def to_parquet(
+        self, prefix_path, dataset="segarray", mode: str = "truncate", compression: Optional[str] = None
+    ):
+        """
+        Save the SegArray object to Parquet. The result is a collection of files,
+        one file per locale of the arkouda server, where each filename starts
+        with prefix_path. Each locale saves its chunk of the object to its
+        corresponding file.
+        Parameters
+        ----------
+        prefix_path : str
+            Directory and filename prefix that all output files share
+        dataset : str
+            Name of the dataset to create in files (must not already exist)
+        mode : str {'truncate' | 'append'}
+            Deprecated.
+            Parameter kept to maintain functionality of other calls. Only Truncate
+            supported.
+            By default, truncate (overwrite) output files, if they exist.
+            If 'append', attempt to create new dataset in existing files.
+        compression : str (Optional)
+            (None | "snappy" | "gzip" | "brotli" | "zstd" | "lz4")
+            Sets the compression type used with Parquet files
+        Returns
+        -------
+        string message indicating result of save operation
+        Raises
+        ------
+        RuntimeError
+            Raised if a server-side error is thrown saving the pdarray
+        ValueError
+            If write mode is not Truncate.
+        Notes
+        -----
+        - Append mode for Parquet has been deprecated. It was not implemented for SegArray.
+        - The prefix_path must be visible to the arkouda server and the user must
+        have write permission.
+        - Output files have names of the form ``<prefix_path>_LOCALE<i>``, where ``<i>``
+        ranges from 0 to ``numLocales`` for `file_type='distribute'`.
+        - 'append' write mode is supported, but is not efficient.
```

Comment:
maybe remove this line since append mode is not supported for segarray

Reply:
Removed.